tidy3d/components/data/data_array.py

```diff
@@ -71,16 +71,23 @@ class DataArray(xr.DataArray):
     _data_attrs: Dict[str, str] = {}
 
     def __init__(self, data, *args, **kwargs):
-        """Initialize ``DataArray``."""
-
-        # initialize with untraced data
-        super().__init__(getval(data), *args, **kwargs)
-        # and put tracers in .attrs
+        super().__init__(data, *args, **kwargs)
         if isbox(data):
             self.attrs[AUTOGRAD_KEY] = data
 
     @property
-    def tracers(self) -> Box:
+    def has_tracers(self) -> bool:
+        return self.tracers is not None
+
+    @property
+    def static(self):
+        vals = self.values if self.values.size == 0 else np.vectorize(getval)(self.values)
+        static_self = self.copy(deep=False, data=vals)
```

Comment:
I think we can use `insert_tracers` here instead. And then modify that method to handle the `attrs.pop(AUTOGRAD_KEY, None)`

We might also want to change the name of that method to reflect that we are not necessarily inserting tracers only. Maybe a more accurate name for it is `.update_with_data()`